assets/sass/_calendar.scss

```diff
@@ -200,3 +200,12 @@ ul.calendar {
 .flex-stretch {
   flex-grow: 1;
 }
+
+.screenreader-only {
+  position: absolute;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+}
```

Comment:
I thought GDS has some sort visually hidden class

Reply:
So it does! ok will push again